src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -328,6 +329,47 @@ internal static void SubscribeSearchForPackagesOption(Step stepInfo, StepUIAutom
                 foundUIElement.Click -= SearchForPackage_Click;
         }
 
+        /// <summary>
+        /// This method will subscribe the Next button from the Step8 Popup for clicking the Package already installed (then it will be expanded).
+        /// </summary>
+        /// <param name="stepInfo">Information about the Step</param>
+        /// <param name="uiAutomationData">Specific UI Automation step that is being executed</param>
+        /// <param name="enableFunction">it says if the functionality should be enabled or disabled</param>
+        /// <param name="currentFlow">The current flow of the Guide can be FORWARD or BACKWARD</param>
+        internal static void SubscribeNextButtonClickEvent(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            var nextbuttonFound = Guide.FindChild((stepInfo.StepUIPopup as PopupWindow).mainPopupGrid, "NextButton") as Button;
+            if (nextbuttonFound == null) return;
+            if(enableFunction)
+                nextbuttonFound.Click += ExecuteAutomaticPackage_Click;
+            else
+                nextbuttonFound.Click -= ExecuteAutomaticPackage_Click;
+                
+        }
+
+        /// <summary>
+        /// This handler will be executed when clicking the next button in the Step 8 Popup so it will be expanding the package content in the LibraryView
+        /// </summary>
+        /// <param name="sender">Next Button</param>
+        /// <param name="e">Event Arguments</param>
+        private static void ExecuteAutomaticPackage_Click(object sender, RoutedEventArgs e)
+        {
+            CollapseExpandPackage(CurrentExecutingStep);
+        }
+
+        /// <summary>
+        /// This method will call the collapseExpandPackage javascript method with reflection, so the package expander in LibraryView will be clicked
+        /// </summary>
+        internal static void CollapseExpandPackage(Step stepInfo)
+        {
+            CurrentExecutingStep = stepInfo;
+            const string jsMethodName = "collapseExpandPackage";
```

Comment:
I believe we have a task about adding unit tests.. We may want to guard in tests to make sure those functions exist

Reply:
@QilongTang then is that ok if I move this js function name to the json file, so we can validate later in a unit test that it exists in the .html ?

Reply:
@QilongTang I've moved the hardcoded strings to the json file so later we can create unit tests that read the json file and check that the js functions exists in the html.
Please check the next commit: https://github.com/DynamoDS/Dynamo/pull/12312/commits/2c84c6eb54517487949f5aef3c5390a4984dc39e